exercises/custom-set/canonical-data.json

```diff
@@ -396,6 +396,16 @@
             "set2": [2, 4]
           },
           "expected": [1, 3]
+        },
+        {
+          "uuid": "20d0a38f-7bb7-4c4a-ac15-90c7392ecf2b",
+          "description": "difference removes all duplicates in the first set",
+          "property": "difference",
+          "input": {
+            "set1": [1, 1],
+            "set2": [1]
+          },
+          "expected": [0]
```

Comment:
Why is the expected set `[0]`? Where does element `0` come from? Shouldn't I expect an empty set as the difference?

Reply:
Yeah, I think that might be a mistake?

Reply:
Yes, that was a mistake. thanks!